chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/ActivityController.java

```diff
@@ -136,27 +127,6 @@ public ActivityMetadataPage metadataForListings(@RequestParam(required = false)
                 ActivityConcept.CERTIFIED_PRODUCT, start, end, pageNum, pageSize);
     }
 
-    @Deprecated
```

Comment:
I can't leave a comment on the line that I want, but I think a number of fields in this class are no longer referenced (atlManager, userManager, resourcePermissions) and are candidates for deleting from this file.